scripts/persona_driven_data_gen/README.md

```diff
@@ -0,0 +1,48 @@
+To start make sure you have your OpenAI and Anthropic API keys and have installed the libraries listed in `requirements.txt`:
+
+```
+pip install -r requirements.txt
+```
+
+This folder contains code to synthetically generate data (both prompts and responses) for target skill using a [persona-driven approach](https://arxiv.org/pdf/2406.20094):
+
+
+**1- Precide Instruction Following:**
+
+```
+# Generate Instruction Following prompts
+python persona_driven_generate_ifdata.py --model "gpt-4o" --start_index 0 --end_index <SAMPLE_SIZE> --output_path if_prompts.jsonl --openai_key Z --org_id YYY --dataset ai2-adapt-dev/personahub_personas --template instruction_following
```

Comment:
can we make the command copy-paste and run? E.g., remove `<SAMPLE_SIZE>` and replace it with a sensible default value. `--openai_key` should be read from the environment possibly? What is this `--org_id`?